src/BuildingBlocks/EventBus/EventBusRabbitMQ/EventBusRabbitMQ.cs

```diff
@@ -97,86 +103,81 @@ public void Publish(IntegrationEvent @event)
                 policy.Execute(() =>
                 {
                     var properties = channel.CreateBasicProperties();
-                    properties.DeliveryMode = 2; // persistent
+                    properties.DeliveryMode = 2;
 
                     _logger.LogTrace("Publishing event to RabbitMQ: {EventId}", @event.Id);
 
                     channel.BasicPublish(
-                        exchange: BROKER_NAME,
-                        routingKey: eventName,
-                        mandatory: true,
-                        basicProperties: properties,
-                        body: body);
+                        BrokerName,
+                        eventName,
+                        true,
```

Comment:
I would not remove parameters name because it is not clear what `true` means in the current context